torch_geometric/nn/functional/bro.py

```diff
@@ -0,0 +1,46 @@
+import torch
+
+
+def bro(x, batch, p=2):
```

Comment:
Can we add type hints to these functions, e.g., `def bro(x: torch.Tensor, batch: Tensor, p: Union[int, str]) -> Tensor:`